### PR TITLE
unpin skylink from dashboard

### PR DESCRIPTION
**Pull request by kwypchlo**
closes https://github.com/SkynetLabs/skynet-webportal/issues/1126

also fixes layout issues - no more scrollbar on long filenames

Screenshot 2021-09-29 at 14 10 22

### CODE REVIEW

**Comment by ruphy** [comment]
this would be such a useful action!!! 😍 thank you for opening this PR, let's hope a review & deploy comes soon

**Review by 7blkampes — APPROVED** [review]
LGTM 👍  can we test it on a server somewhere? changes look good though

**Comment by kwypchlo** [comment]
we have one backend change that needs to be merged before we can ship this https://github.com/SkynetLabs/skynet-accounts/pull/75

**Comment by ruphy** [comment]
that change was merged, can't wait for this 👍
thanks for all of your work!!

**Review by ro-tex — APPROVED** [review]
LGTM.

File: packages/dashboard/src/services/authServerSideProps.js [hunk 1]
```diff
@@ -5,6 +5,14 @@ const isProduction = process.env.NODE_ENV === "production";
 export default function authServerSideProps(getServerSideProps) {
   return function authenticate(context) {
     if (isProduction && (!("ory_kratos_session" in context.req.cookies) || !("skynet-jwt" in context.req.cookies))) {
+      if ("skynet-jwt" in context.req.cookies) {
```
Comment (peterjan) [inline packages/dashboard/src/services/authServerSideProps.js:8]: I understand it's probably not that important, but this in fact prints wrong information in the case where both fields are **not** present in the cookies. 

That would print: 
`Cookie skynet-jwt is present but ory_kratos_session is not - redirecting to login`
`Cookie ory_kratos_session is present but skynet-jwt is not - redirecting to login`

```
const requiredCookiesSet = "ory_kratos_session" in context.req.cookies && "skynet-jwt" in context.req.cookies
if (isProduction && !requiredCookiesSet) {
    if (!"skynet-jwt" in context.req.cookies) {
        console.log(`Cookie "skynet-jwt" is required but not set - redirecting to login`);
    }
    if (!"ory_kratos_session" in context.req.cookies) {
        console.log(`Cookie "ory_kratos_session" is required but not set - redirecting to login`);
    }
    ...
}
```

**Review by peterjan — APPROVED** [review]

**Review by peterjan — APPROVED** [review]

**Review by ro-tex — APPROVED** [review]

**Pull request merged**